packages/storage-plus/src/map.rs

```diff
@@ -108,13 +110,29 @@ where
     }
 }
 
+#[cfg(feature = "iterator")]
+impl<'a, K, T> Map<'a, K, T>
+where
+    T: Serialize + DeserializeOwned,
+    K: PrimaryKey<'a>,
+{
+    #[cfg(feature = "iterator")]
+    pub fn sub_prefix_de(&self, p: K::SubPrefix) -> Prefix<T, K::SuperSuffix> {
```

Comment:
This prefix and suffix (sub prefix and subsuffix) stuff looks proper

Reply:
This is coolest thing I did in all this I guess.